frontend/src/views/IntegrationsPage/components/IntegrationsSection/IntegrationsSection.tsx

```diff
@@ -34,6 +38,9 @@ export const IntegrationsSection = ({
   const { popUp, handlePopUpOpen, handlePopUpClose, handlePopUpToggle } = usePopUp([
     "deleteConfirmation"
   ] as const);
+
+  const { currentWorkspace } = useWorkspace();
+  const { data: bot } = useGetWorkspaceBot(currentWorkspace?._id ?? "");
```

Comment:
Can you tell me why this call is done, if its already available in top page component

Reply:
updated the component to get those values as a prop 👍🏼